expression-language/3.0/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Jakarta Expression Language Specification 3.0"
+date: 2019-07-22T16:45:02+0200
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Expression Language defines an expression language for Java applications
+
+* [Jakarta Expression Language 3.0 Specification Document](./expression-language_3.0.pdf) (PDF)
+* [Jakarta Expression Language 3.0 Specification Document](./expression-language_3.0.html) (HTML)
+* [Jakarta Expression Language 3.0 Javadoc](./apidocs)
+* Maven coordinates
+  * [jakarta.el:jakarta.el-api:jar:3.0.3](https://search.maven.org/artifact/jakarta.el/jakarta.el-api/3.0.3/jar)
+
+# Ballots
+
+## Release Review
+
+
+|                       |  Yes    | No      | Abstain  |
+|-----------------------|---------|---------|----------|
+|Fujitsu                |         |         |          |
+|IBM                    |         |         |          |
+|Oracle                 |         |         |          |
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
+
+# Compatible Implementations
+
+* Expression Language RI 3.0.3
```

Comment:
There are no longer "Reference Implementations", so this Compatible Implementation
needs to use something other than "RI".  If you don't have a clever project name,
consider something like "EE4J Implementation of Expression Language".

Also, this should include a link to the implementation project.